src/inputs/Table/ColumnTypes.js

```diff
@@ -141,6 +141,9 @@ export const getDefaultColumnsProperties = (onCellChange) => {
     onCellValueChanged: (event) => {
       onCellChange(event);
     },
+    cellClass: (params) => {
+      return params.column.colDef.editable ? null : classes.nonEditableCell;
```

Comment:
Are we sure the hole chaining of `params.column.colDef.editable` is defined ? Maybe we should use the optional chaining operator (`?.`) to avoid errors here.